tests/test-definitions.txt

```diff
@@ -0,0 +1,156 @@
+# test_suite [flags ...] -- [additional-params...]
+# for possible flags and params see
+# scripts/generateJenkinsScripts.py --help-flags
+
+
+# Single Server only Tests
+BackupAuthNoSysTests single
+BackupAuthSysTests single
+BackupNoAuthNoSysTests single
+BackupNoAuthSysTests single
+
+# Upgrade tests
+upgrade_data_3.2.* weight=1000 single
+upgrade_data_3.3.* weight=1000 single
+upgrade_data_3.4.* weight=1000 single
+upgrade_data_3.5.* weight=1000 single
+upgrade_data_3.6.* weight=1000 single
+upgrade_data_3.7.* weight=1000 single
+
+# Single Server Replication Tests
+replication_fuzz weight=500 single -- --extraArgs:log.level replication=trace
+replication_ongoing weight=500 single -- --extraArgs:log.level replication=trace
+replication_ongoing_frompresent single -- --extraArgs:log.level replication=trace
+replication_ongoing_global weight=500 single -- --extraArgs:log.level replication=trace
+replication_ongoing_global_spec single -- --extraArgs:log.level replication=trace
+replication_static weight=500 single -- --extraArgs:log.level replication=trace
+replication_sync weight=500 single -- --extraArgs:log.level replication=trace
+shell_replication single
+
+# Misc single server tests
+active_failover single
+agency weight=350 single -- --dumpAgencyOnError true
+agency-restart weight=250 single
+arangobench weight=1000 single
+arangosh single
+authentication weight=1000 single
+endpoints single -- --skipEndpointsIpv6 true
+paths_server weight=500 single
+permissions weight=500 single
+recovery weight=2000 buckets=4 single
+version single
+
+# Shell client tests Single Server
+shell_client weight=500 single suffix=http -- $EncryptionAtRest
+shell_client weight=500 single suffix=http2 -- --http2 true
+shell_client weight=500 single suffix=vst -- --vst true
+
+shell_client_aql weight=500 single suffix=http
+shell_client_aql weight=500 single suffix=http2 -- --http2 true
+shell_client_aql weight=500 single suffix=vst -- --vst true
+shell_client_aql weight=500 single enterprise suffix=encrypt -- --encryptionAtRest true
+
+shell_server_aql weight=750 single buckets=5
+# execute without buckets in single server, but use 5 buckets in cluster
+shell_server weight=1000 single
+
+# Catch tests are executed in single env
+catch weight=1000 single
```

Comment:
We don't use catch anymore, so let's rename the test definition. The suite name "gtest" is understood by our test framework already.
```suggestion
# C++ unit tests are executed in single env
gtest weight=1000 single
```

Reply:
I suggest we also add @arangodb/team-qa to the CODEOWNERS file for `tests/test-definitions.txt`, so it will be mandatory that they review changes to this file.